composer/callbacks/generate.py

```diff
@@ -46,7 +46,10 @@ def __init__(self,
     def run_event(self, event: Event, state: State, logger: Logger) -> None:
         if state.get_elapsed_duration() is not None and self.check_interval(
                 state, event) and self.last_generate_batch != state.timestamp.batch:
+            start = time.time()
             self.generate(state, logger)
+            diff = time.time() - start
+            log.info('Generate callback ran in %d seconds for %d prompts', diff, len(self.prompts))
```

Comment:
lint correctly identified that you are missing the instantiation of `log` and the import of `time`